src/devices/Bmxx80/Bme680.cs

```diff
@@ -150,17 +346,54 @@ public async Task<double> ReadPressureAsync()
         /// <returns>Calculated temperature.</returns>
         public Task<Temperature> ReadTemperatureAsync()
```

Comment:
I have mixed feelings if this should be returning NaN equivalen or perhaps it would be better to do `bool TryReadTemperature(out Temperature temperature)`

Reply:
@joperezr had a similar thought on this, would you prefer returning the last measured result or the TryReadTemperature method?

Reply:
TryRead or NaN both sound good. I'll need to have some conversations internally what is preferred (I suspect TryRead will be the guidelines)